Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
```

Comment:
can we test filters for this parser using ASC portal?

Reply:
done